tests/baselines/reference/es6ExportEqualsInterop.errors.txt

```diff
@@ -190,13 +190,13 @@ tests/cases/compiler/main.ts(106,15): error TS2498: Module '"class-module"' uses
              ~
 !!! error TS2305: Module '"function"' has no exported member 'a'.
                             ~~~~~~~~~~
-!!! error TS2497: Module '"function"' resolves to a non-module entity and cannot be imported using this construct.
+!!! error TS2497: This module can only be imported with ECMAScript imports by turning on the 'esModuleInterop' flag and using a default import.

```

Comment:
Damn it, okay, last time and I swear I'l pull it in no matter what.

```
This module can only be referenced with ECMAScript imports/exports by turning on the '{0}' flag and referencing its default export.
```

Reply:
@DanielRosenwasser changed. +1:

Reply:
@DanielRosenwasser changed. :+1:

Reply:
@DanielRosenwasser ok :) :+1: